demo-app/src/androidTest/java/com/worldpay/access/checkout/sample/testutil/UITestUtils.kt

```diff
@@ -143,4 +145,11 @@ object UITestUtils {
             .check(matches(isDisplayed()))
             .perform(NavigationViewActions.navigateTo(fragmentId))
     }
+
+    fun retrieveEnteredText(accessEditText: AccessEditText): String {
+        val property = accessEditText.javaClass.getDeclaredField("editText")
+        property.isAccessible = true
+        val editText = property.get(accessEditText) as EditText
```

Comment:
Do you mind putting the property to non accessible once you have extracted the text?